contentcuration/contentcuration/views/public.py

```diff
@@ -71,3 +73,17 @@ def get_channel_name_by_id(request, channel_id):
         return HttpResponseNotFound('Channel with id {} not found'.format(channel_id))
     return HttpResponse(json.dumps({"name": channel.name, "description": channel.description, "version": channel.version}))
 
+
+class InfoViewSet(viewsets.ViewSet):
```

Comment:
Can you add docstrings on what this viewset is for?

Reply:
And/or link to the spec doc:
https://docs.google.com/document/d/1XKXQe25sf9Tht6uIXvqb3T40KeY3BLkkexcV08wvR9M/edit#